### PR TITLE
Updated mutator docs

### PR DESCRIPTION
**Pull request by psfinaki**
A few mutator related updates:
1. Fixed broken links in the docs
2. Renamed mutations to mutators in some places (I got lazy at some point)
3. Add the names of the mutations alongside the names of the mutators. I've been missing this myself and the names are not always obvious

### CODE REVIEW

**Comment by rouke-broersma** [comment]
I think we actually wanted to standardize on Mutations instead of Mutators but I'm not 100 percent sure. @richardwerkman ?

File: docs/mutators.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 ---
```
Comment (richardwerkman) [inline docs/mutators.md:1]: Renaming this file will break the website. I'd suggest keeping this mutations for now as these docs contain both the mutators and the specific mutations

**Review by richardwerkman — CHANGES_REQUESTED** [review]
It's good to have the actual names for the ignore functionality in the docs! Thanks

File: docs/mutators.md (hunk 1 above)
Comment (psfinaki) [reply docs/mutators.md]: Alright. I removed all the renamings, only left the _custom_edit_url_ change in that regard, I guess it should fix this issue here:
image

**Review by richardwerkman — APPROVED** [review]

**Pull request merged**